src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/DataGrid.xaml

```diff
@@ -789,10 +789,10 @@
         <Setter Property="SnapsToDevicePixels" Value="True" />
     </Style>
 
-    <Style TargetType="{x:Type DataGridCheckBoxColumn}">
+    <!--<Style TargetType="{x:Type DataGridCheckBoxColumn}">
         <Setter Property="ElementStyle" Value="{StaticResource DataGridCheckBoxElementDefaultStyle}" />
         <Setter Property="EditingElementStyle" Value="{StaticResource DataGridCheckBoxEditingElementDefaultStyle}" />
-    </Style>
+    </Style>-->
```

Comment:
Why is this commented out instead of removed? Imho it should at least have a note explaining why is this commented out if it ain't removed.

Reply:
Hey @MitchRazga , you can go ahead and remove them. In the PR that I have created for .NET 9, I removed it.
Moreover, you need to run `.\ThemeGenerator.Fluent.ps1` script as well to regenerate the Fluent.Light.xaml, Fluent.Dark.xaml and Fluent.HC.xaml files.